### ISSUE TITLE
ensure new micropurchases are Section 889 compliant

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#939]
## Background Information

Follow-up to https://github.com/18F/tts-tech-portfolio/issues/905. From @michellepetryszyn [in Slack](https://gsa-tts.slack.com/archives/G017C3R27MM/p1602267919002300):

> We all should have received a document to sign in Docusign regarding pcard purchases and Section 889. Per this new guidance, we have to prepare an open market justification if we are not buying from existing contract vehicles. I put together [this template](https://docs.google.com/document/d/1jRMwRi94_O_YJ_0JkvKtRhniitjU2ZMFggXRirL6eOE/edit#heading=h.mw15bdywiu3d) to help us with this new requirement. Also, per the guidance this needs to be signed by your AO before the purchase. … it is for all IT buys

See also:

- [SmartPay Bulletin 33](https://smartpay.gsa.gov/sites/default/files/SP%20SmartBulletin%20No.%20033%20-%20Sect%20889%20Prohibited%20Telecomms%20(PART%20B).pdf)

## Implementation Steps

- [x] Determine division of labor for this issue between Tech Portfolio, BizOps, and Acquisition
- [ ] Update documentation ([Handbook](https://handbook.tts.gsa.gov/purchase-requests/#behind-the-scenes) and/or other relevant docs)
- [ ] Build into spreadsheet flow

## Acceptance Criteria

- [ ] The Open Market Justification is incorporated into the p-card purchase process

**Comment by melanienleopold** [comment]
@michellepetryszyn and I are taking the lead on this. We will add a section to the micropurchase form that will cover this. More meetings to come with the larger pcard team to inform them on the process.

**Comment by melanienleopold** [comment]
[Test Google doc](https://docs.google.com/forms/d/1pWL4L-wabM_wVL_EIdUDn-dTORkidOjyKyElCeCo9Lk/edit) with 889 information included.

**Comment by afeld** [comment]
Waiting on p-card meeting next week.

**Comment by melanienleopold** [comment]
Consensus on 889. It is built into the new Google form that will be updated on 1/22.

**Comment by its-a-lisa-at-work** [comment]
@melanienleopold I'm adding myself to this because of the activity that I'm doing at the top level for SCRM and Section 889 -- there is a tool that is set up to help us with this process, so I want to jump in to see where in the process that would fit.

**Comment by melanienleopold** [comment]
Closing in favor of [#1229](https://github.com/18F/tts-tech-portfolio/issues/1229)
